### PR TITLE
Update to the latest pack

### PR DESCRIPTION
**Pull request by scothis**
- consume latest pack cli
- consume latest packs/* images
- include stack metadata in builder.toml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 language: go
 go:
-  - '1.11.x'
-sudo: required
-services:
-  - docker
```
Comment (ericbottard) [inline .travis.yml]: Isn't that required? Or do we always get a docker daemon?

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module github.com/scothis/riff-buildpack-group
```
Comment (ericbottard) [inline go.mod]: I'm not sure I follow what this is for?
Is there some go code that is being compiled for the builder itself (I know the ITs is a separate module)

File: .travis.yml (hunk 1 above)
Comment (scothis) [reply .travis.yml]: sudo and docker are both provided by default, so there's no need to specify them.

File: go.mod (hunk 2 above)
Comment (scothis) [reply go.mod]: It shouldn't be there.

**Pull request merged**